sdk/python/test/local_execution/local_execution_test.py

```diff
@@ -244,11 +252,32 @@ def setup_and_teardown(self):
     @pytest.mark.parametrize(
         'test_data', docker_specific_pipeline_funcs, ids=idfn)
     def test_execution(self, test_data: TestData):
-        if test_data.pipeline_func_args is not None:
-            pipeline_task = test_data.pipeline_func(
-                **test_data.pipeline_func_args)
+        if test_data.name == 'Importer Workspace':
+            ws_root = f'{ws_root_base}_docker'
+            pipeline_root = f'{pipeline_root_base}_docker'
+            if os.path.isdir(ws_root):
+                shutil.rmtree(ws_root, ignore_errors=True)
+            Path(ws_root).mkdir(parents=True, exist_ok=True)
+            local.init(
+                runner=local.DockerRunner(user=f'{os.getuid()}'),
```

Comment:
Can we enhance TestData to accept a new optional arg of `docker_runner_args` to make this reusable?